src/Writer/MarkdownWriter.php

```diff
@@ -47,18 +49,20 @@ public function createDocumentation(string $filepath, array $installedPackages,
                 }
             }
 
-            // @TODO: Maybe add documentation for packages who were documented but not installed (anymore)
-
             $documentation[] = "";
+        }
 
-            $handle = @fopen($filepath, "w");
+        // @TODO: Maybe add documentation for packages who were documented but not installed (anymore)
```

Comment:
I guess I missspelled this, should probably be "packages *which* were documented"